### ISSUE TITLE
 Panel object don't stay at the bottom of a vertical panel 

### ISSUE THREAD

**Issue by elcste** [mate-desktop/mate-panel#763]
#### Expected behaviour

Items I move to the bottom of a vertical panel should stay there, just like they stay at the right end of the horizontal panel.

#### Actual behaviour

When I log in or reload the panel layout with MATE Tweak, any objects set to be at the bottom move up to be just below the items at the top. It's getting boring to move them back down each time I log in :-P

#### Steps to reproduce the behaviour

My preferred layout is like this:

> Brisk
> Dock
> 
> (empty space)
> 
> Notification Area
> Indicators
> Clock

But every time I log in or load the panel the empty space disappears and everything is aligned to the top.

This worked fine in all previous versions of MATE I've used.

It doesn't matter if I start with a blank panel and add everything or take an existing one and modify it. It's not specific to these items. I've tried other combinations, including the inverse and without the dock applet.

Not sure if it matters, but in the panel layout file the items that should be at the bottom do have "panel-right-stick=true".

#### MATE general version
1.20.1

#### Package version
mate-panel 1.20.1-0ubuntu1

#### Linux Distribution

Ubuntu MATE Bionic

#### Link to downstream report of your Distribution

https://bugs.launchpad.net/ubuntu/+source/mate-panel/+bug/1762582

Thanks!

**Comment by lukefromdc** [comment]
I just tested this, found it happening some of the time but not all of the time. Showing or removing the panel hide buttons would move all applets on a left side panel-and if they were showing restarting the panel ALWAYS moved a single applet set on the bottom to the top.

I also found a workaround, though as I normally use a bottom panel it only got a brief test. Put a separator on the vertical panel, then move it up as far as you can to use up all that space above the applets that apparently is not getting saved. Then the rest of the applets should hopefully stay where they are relative to the spacer. Worked for me, give it a try.

I am guessing the data is sometimes getting written as distance from left instead of distance from top, but spacing between applets must be OK if this workaround always works.  I can see how "right stick" would become meaningless on a vertical panel if it does not treat "bottom" as "right" or no longer does so. At any rate, applets should stay where you put them unless forced over by expansion of another applet. I do recall that GNOME 2 of old used to have issues with applets that moved around, crashed, or did not always appear. Marking this one confirmed

**Comment by info-cppsp** [comment]
duplicate of https://github.com/mate-desktop/mate-panel/issues/745

**Comment by elcste** [comment]
@lukefromdc Thanks for the confirmation. I tried the workaround with the separator and it didn't seem to work for me, but I also only tried it briefly.

@info-cppsp 's workaround in #745 does work me, however. (Obrigado!) Specifically, if I move my bottom applets just one pixel up, so that nothing it touching the bottom edge, then they seem to stay put. I lose the Fitt's law advantage of the corner for the clock, but it's much better than moving everything on every log in ;-).

I tried editing my saved layout file  (with applets at the bottom edge) to set the `right-stick` line to be false for everything, but this still doesn't work. It seems like it only works in 1.20 with the `position` set to not be the bottom edge.
